es-ds-docs/pages/molecules/skeleton.vue

```diff
@@ -0,0 +1,167 @@
+<script setup lang="ts">
+const loading = ref(true);
+
+const asyncTimeout = async (seconds = 3) => {
+    const millisecondTimeout = seconds * 1000;
+    return new Promise((resolve) => {
+        setTimeout(resolve, millisecondTimeout);
+    });
+};
+
+const startLoading = async () => {
+    loading.value = true;
+    await asyncTimeout();
+    loading.value = false;
+};
+
+const esSkeletonProps = [
+    ['animation', 'String', 'wave', `Options are 'wave', 'fade', or 'none'`],
+    ['height', 'String', '1rem', `Manually set height`],
+    ['width', 'String', 'auto', `Manually set width`],
+    ['size', 'String', 'null', `Manually set width and height, overriding 'height' and 'width' props`],
+];
+
+const esSkeletonImgProps = [
+    ['aspect', 'String', '16:9', `Adds a container around the skeleton image with given aspect ratio`],
+    ['noAspect', 'Boolean', 'false', `Override the default aspect ratio and height, width, or size`],
+    ...esSkeletonProps,
+];
```

Comment:
Could throw a `.sort((a, b) => a[0].localeCompare(b[0]))` here to sort these alphabetically